Parser/Device/DeviceParserAbstract.php

```diff
@@ -223,6 +224,7 @@
         'ME' => 'Metz',
         'MX' => 'MEU',
         'MI' => 'MicroMax',
+        'MIX' => 'MIXC',
```

Comment:
Please use only 2 characters as shortcode. If useful shortcodes are already taken use something like 'M1', 'M2'
(Piwik isn't able to handle longer short codes)